packit_service/worker/build/copr_build.py

```diff
@@ -256,21 +259,47 @@ def run_build(
                 for field, (old, new) in ex.fields_to_change.items():
                     table += f"| {field} | {old} | {new} |\n"
 
+                boolean_note = ""
+                if "unlisted_on_hp" in ex.fields_to_change:
+                    boolean_note += (
+                        "The `unlisted_on_hp` field is represented as `list_on_homepage`"
+                        " in the packit config."
+                        "By default we create projects with `list_on_homepage: False`.\n"
+                    )
+
+                if "delete_after_days" in ex.fields_to_change:
+                    boolean_note += (
+                        "The `delete_after_days` field is represented as `preserve_project`"
+                        " in the packit config (`True` is `-1` and `False` is `60`)."
+                        "By default we create projects with `preserve: True` "
+                        "which means `delete_after_days=60`.\n"
+                    )
```

Comment:
Unfortunately, this will not be used with the change above. (We don't have permission error for **our** projects and we don't update those values for **custom** projects.)

I'm leaving it there for the future if we change our minds with that condition since it's really tricky.